lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -1926,11 +1939,158 @@
         return DateTimeFormat;
     })();
 
+    const PluralRules = (function() {
+        if (InitType !== "Intl") {
+            return;
+        }
+
+        /**
+         * Initializes the given pluralRules object
+         *
+         * ECMA 402: #sec-initializepluralrules
+         *
+         * @param {Object} pluralRules
+         * @param {String|String[]} locales
+         * @param {Object} options
+         */
+        const InitializePluralRules = function (pluralRules, locales, options) {
+            const requestedLocales = CanonicalizeLocaleList(locales);
+            options = options === undefined ? _.create() : Internal.ToObject(options);
+            const opt = _.create();
+            opt.matcher = GetOption(options, "localeMatcher", "string", ["lookup", "best fit"], "best fit");
+            pluralRules.type = GetOption(options, "type", "string", ["cardinal", "ordinal"], "cardinal");
+
+            SetNumberFormatDigitOptions(pluralRules, options, 0, 3);
+
+            // %PluralRules%.[[RelevantExtensionKeys]] = [] (#sec-intl.pluralrules-internal-slots)
+            const r = ResolveLocale(platform.isPRLocaleAvailable, requestedLocales, opt, []);
+
+            pluralRules.locale = r.locale;
+            pluralRules.pluralCategories = platform.pluralRulesKeywords(pluralRules);
+
+            pluralRules.initializedPluralRules = true;
+
+            return pluralRules;
+        };
+
+        /**
+         * Returns a String value representing the plural form of n according to
+         * the effective locale and the options of pluralRules
+         *
+         * ECMA 402: #sec-resolveplural
+         *
+         * @param {Object} pluralRules
+         * @param {Number} n
+         */
+        const ResolvePlural = function (pluralRules, n) {
+            if (!_.isFinite(n)) {
+                return "other";
+            }
+
+            return platform.pluralRulesSelect(pluralRules, n);
+        };
+
+        const PluralRules = function (locales = undefined, options = undefined) {
```

Comment:
> = undefined [](start = 45, length = 12)

unsupplied params are undefined by default, we don't need to specify it

Reply:
This raises the point that this is a by-design "anti-pattern" and probably warrants a comment per-occurrence explaining why. (to wit: "// use `= undefined` on formals to force the `length` of the function to be 0, per spec")